linode/domain/resource_test.go

```diff
@@ -68,8 +67,8 @@ func TestAccResourceDomain_basic_smoke(t *testing.T) {
 					resource.TestCheckResourceAttrSet(resName, "retry_sec"),
 					resource.TestCheckResourceAttrSet(resName, "expire_sec"),
 					resource.TestCheckResourceAttrSet(resName, "status"),
-					resource.TestCheckNoResourceAttr(resName, "master_ips"),
-					resource.TestCheckNoResourceAttr(resName, "axfr_ips"),
+					resource.TestCheckResourceAttr(resName, "master_ips.#", "0"),
+					resource.TestCheckResourceAttr(resName, "axfr_ips.#", "0"),
```

Comment:
This seems like a breaking change? Will this framework resource work well with the state created by the previous SDKv2 resource? Did we test that?

Reply:
I believe this is because now we have to set State to a specific null value instead of letting it be empty.